app/models/reports/instrument_access_list_export_raw.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Reports
+
+  class InstrumentAccessListExportRaw
+    include DateHelper
+    include TextHelpers::Translation
+
+    def initialize(product_users)
+      @product_users = product_users
+    end
+
+    def to_csv
+      CSV.generate do |csv|
+        csv << headers
+        @product_users.each do |product_user|
+          csv << build_row(product_user)
+        end
+      end
+    end
+
+    def filename
+      "instrument_access_list.csv"
```

Comment:
How about we include the product name here:
```suggestion
      "#{product.name}_access_list.csv"
```
You could pass in the product on initialization.